src/main/java/spireMapOverhaul/SpireAnniversary6Mod.java

```diff
@@ -208,6 +209,7 @@ public void receiveEditCards() {
     @Override
     public void receivePostInitialize() {
         initializedStrings = true;
+        unfilteredAllZones.forEach(AbstractZone::loadStrings);
```

Comment:
As a funny consequence of how all the strings stuff is set up, the zones in `unfilteredAllZones` weren't getting their strings loaded before I added this. So if you wrote natural code that looked up a zone in this list and tried to get its name, it would be null, and you would be very confused (like I was). Best to not have traps like that.

Code like the map display worked due to calling `copy` first, since that made a new instance and loaded up strings again.